winpr/libwinpr/utils/wlog/ConsoleAppender.c

```diff
@@ -138,6 +138,7 @@ static BOOL WLog_ConsoleAppender_WriteMessage(wLog* log, wLogAppender* appender,
 
 	if (message->Level != WLOG_OFF)
 		fprintf(fp, "%s%s\n", message->PrefixString, message->TextString);
+	fflush(fp);
```

Comment:
Hum this one seems unrelated.
Doesn't it have the possibility to kill the performances of the console appender ?

Reply:
oops ^^